Addbackground image and responsive

nuriyealp opened the pull request:
## Description
<!-- What does this code change? Why did I choose this approach? Did I learn anything worth sharing? -->
I added an image to the landing page and I made the page responsive for tablet.

## Resouce links or research articles
<!-- What did I learn anything worth sharing like links or sites which I researched ? -->
https://stackoverflow.com/questions/60682426/material-ui-adds-padding-to-the-body-tag-when-a-dialog-is-opened/74577365#74577365
## Related Issue
Closes #112 

<!-- If you write "closes" followed by the Github issue number, it will automatically close the issue for you when the PR merges -->

## Acceptance Criteria
Add an image to the landing page
Make the page responsive to tablet

<!-- Include AC from the Github issue -->

## Type of Changes

<!-- Put an `✓` for the applicable box: -->

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

<!-- If UI feature, take provide screenshots -->


### After

<!-- If UI feature, take provide screenshots -->
<img width="1440" alt="Screen Shot 2022-11-25 at 7 35 37 PM" src="https://user-images.githubusercontent.com/88150284/204071010-e7e61eb6-05d8-4839-9668-ca197401d963.png">


## Testing Steps / QA Criteria

<!-- Provide steps the other team members and mentors need to follow to properly test your additions. -->

elenamagay reviewed (changes requested):
The background looks good but when I switch to tablet or narrower screen there is a big gap between the description and Sign in. Also, please resolve error in the console log. Thanks

nuriyealp commented:
@elenamagay Thank you.
I made the gap smaller and resolved the error in the console log.

XeniyaDob reviewed (changes requested):
Browser screen:
The description and sign-in form are too wide, in my opinion.
<img width="2537" alt="Screen Shot 2022-11-29 at 9 45 24 AM" src="https://user-images.githubusercontent.com/11164899/204561050-bae6e320-9608-4384-8a6d-f1abce43e1a6.png">

Could you keep this width and center the content?
<img width="2438" alt="Screen Shot 2022-11-29 at 9 46 48 AM" src="https://user-images.githubusercontent.com/11164899/204561482-dc74b82f-0651-43d7-9197-3f5e6442cb2b.png">

Ipad and mobile screens:  The content of the page should be centered. There is no margin on the top. 
<img width="550" alt="Screen Shot 2022-11-29 at 9 44 53 AM" src="https://user-images.githubusercontent.com/11164899/204561098-a35a2cda-9ce3-4742-bd7b-9a05111aae22.png">

nuriyealp commented:
I added the margin top of the page. The content of the page is centered now.
I adjusted the picture. The description and sign-in form are not too wide on the Browser screen now.

XeniyaDob reviewed (approved):
Great job, thank you!

elenamagay commented on src/components/Layout/LayoutPublic.js:
```diff
@@ -4,7 +4,7 @@ import { Outlet } from "react-router-dom";
 
 import FooterPublic from "../Footer/FooterPublic";
-const LayoutPublic = ({ children }) => {
+const LayoutPublic = ({ children, theme }) => {
```
why did you declare theme here if you never use it in this file?
